CIP-0001/README.md

```diff
@@ -1,185 +1,374 @@
+Rationale      | The rationale fleshes out the specification by describing what motivated the design and what led to particular design decisions. It should describe alternate designs considered and related work. The rationale should provide evidence of consensus within the community and discuss significant objections or concerns raised during the discussion. <br/><br/>It must also explain how the proposal affects the backward compatibility of existing solutions when applicable. If the proposal responds to a [CPS][], the 'Rationale' section should explain how it addresses the CPS and answer any questions that the CPS poses for potential solutions.
+Path to Active | Organised in two sub-sections:<br/><h5>Acceptance Criteria</h5>Describes what are the acceptance criteria whereby a proposal becomes _'Active'_.<br/><h5>Implementation Plan</h5>A plan to meet those criteria. Or `N/A` if not applicable.
 Copyright      | The CIP must be explicitly licensed under acceptable copyright terms ([see below](#licensing)).
 
 <details>
   <summary><span id="header-preamble"><strong>Header preamble</strong></span></summary>
 
-Each CIP must begin with an [RFC 822](https://www.ietf.org/rfc/rfc822.txt) key:value style header preamble (also known as 'front matter data'), preceded and followed by three hyphens (`---`).
+Each CIP must begin with a YAML key:value style header preamble (also known as 'front matter data'), preceded and followed by three hyphens (`---`).
 
-Field     | Description
----       | ---
-`CIP`     | CIP number (without leading 0), or "\?" before being assigned
-`Title`   | A succinct and descriptive title
-`Authors` | Comma-separated list of authors' real names and email addresses (e.g. John Doe <john.doe@email.domain>).
-`Status`  | Draft \| Proposed \| Active \| Obsolete
-`Type`    | Core \| Process \| Informational
-`Created` | date created on, in ISO 8601 (YYYY-MM-DD) format
-`License` | abbreviation of an approved license(s)
+Field          | Description
+---            | ---
+`CIP`          | The CIP number (without leading 0), or "\?" before being assigned
+`Title`        | A succinct and descriptive title
+`Status`       | Proposed \| Active \| Inactive (.._reason_..)
+`Category`     | One of the registered categories covering one area of the ecosystem.
+`Authors`      | A list of authors' real names and email addresses (e.g. John Doe <john.doe@email.domain>)
+`Implementors` | A list of implementors committed to delivering an implementation of the proposal, when applicable. `N/A` when not applicable and `[]` when there's currently no implementor.
+`Discussions`  | A list of links where major technical discussions regarding this CIP happened. Links should include any discussion before submission, and _must_ include a link to the pull request that created the CIP and any pull request that modifies it.
+`Solution-To`  | A list of [CPS][] that this CIP addresses, if any. Omitted when not applicable.
```

Comment:
I originally put addresses there, and thought it might be confusing with the other meaning of addresses in our space. People might misinterpret that as a kind of donation section or whatever. I preferred to avoid the term altogether.